model/registry/gen-ai.yaml

```diff
@@ -0,0 +1,87 @@
+groups:
```

Comment:
Sometimes the response will have response fields related to malicious prompts. 

The response looks different that normal responses. For example with openai it may have a `code` field with a value `ResponsibleAIPolicyViolation` or a `content_filter_result` field that has reasons for why the content was filtered. 

Can we standardize how these LLM security fields are sent?

Reply:
This is a good idea. Can we add this with a follow-up PR once our MVP here gets merged? I added a [backlog item here](https://github.com/orgs/open-telemetry/projects/82/views/1?pane=issue&itemId=59914133) to track it.